ui/src/features/Editor/components/RightPanel/components/VersionHistory/index.tsx

```
@@ -36,40 +42,45 @@ const VersionHistoryList: React.FC<Props> = ({ versionHistory }) => {
+        {isFetching ? (
+          <LoadingSkeleton className="pt-12" />
+        ) : previousVersions && previousVersions.length > 0 ? (
+          <div className="flex flex-col overflow-auto">
+            {previousVersions?.map((version) => (
+              <Version
+                // key={version}
+                version={version}
+                isSelected={version.version === selectedProjectSnapshotVersion}
+                onClick={() => handleVersionClick(version.version)}
+                onDoubleClick={handleDoubleClick}
+              />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)